cmd/vicadmin/server.go

```diff
@@ -47,10 +53,26 @@ const (
 	formatZip
 )
 
+const (
+	sessionExpiration      = time.Hour * 24
+	sessionCookieKey       = "sessiondata"
+	sessionCreationTimeKey = "created"
+	sessionUsernameKey     = "username"
+	loginPagePath          = "/authentication"
+	authFailure            = loginPagePath + "?unauthorized"
+)
+
 func (s *server) listen() error {
 	defer trace.End(trace.Begin(""))
 
 	var err error
+	s.uss = NewUserSessionStore()
+	s.uss.cookies.Options = &sessions.Options{
+		Path:     "/",
+		MaxAge:   3600 * 24,
```

Comment:
Yeah I gotta go through and replace this timeout in a bunch of places w/ the same constant.